packages/ui/src/stubs/datamapper/data-mapper.ts

```diff
@@ -92,6 +92,10 @@ export const conditionalMappingsToShipOrderJsonXslt = fs
   .readFileSync(path.resolve(__dirname, './json/ConditionalMappingsToShipOrderJson.xsl'))
   .toString();
 
+export const orgXsd = fs.readFileSync(path.resolve(__dirname, './xml/Org.xsd')).toString();
+export const contactsXsd = fs.readFileSync(path.resolve(__dirname, './xml/Contacts.xsd')).toString();
+export const orgToContactsXslt = fs.readFileSync(path.resolve(__dirname, './xml/OrgToContacts.xsl')).toString();
```

Comment:
What do you think of reading these files using the promise API, inside of the describe method? This way, importing this module won't automatically read some files synchronously

Reply:
I'm interested in, how it looks like? just `export const orgXsdPromise = fs.readFileAsync()` and each test does `await orgXsdPromise`?